administrator/components/com_workflow/Field/WorkflowStateField.php

```diff
@@ -0,0 +1,80 @@
+<?php
+/**
+ * @package     Joomla.Administrator
+ * @subpackage  com_workflow
+ *
+ * @copyright   Copyright (C) 2005 - 2017 Open Source Matters, Inc. All rights reserved.
+ * @license     GNU General Public License version 2 or later; see LICENSE.txt
+ */
+namespace Joomla\Component\Workflow\Administrator\Field;
+
+defined('JPATH_BASE') or die;
+
+use Joomla\CMS\Factory;
+use Joomla\CMS\Form\FormHelper;
+
+FormHelper::loadFieldClass('groupedlist');
```

Comment:
Could perhaps (??) be replaced with a namespace "use"? (or is it autoloaded anyways?)